modules/installation-custom-aws-vpc.adoc

```diff
@@ -206,6 +242,9 @@ h|Reason
 * `AWS::EC2::SubnetRouteTableAssociation`
 2+|Your VPC can have private subnets. The provided CloudFormation templates
```

Comment:
I know you didn't add this line, but where is the "provided" templates?

Reply:
@ogradyp can you please advise on this?
This is not new text, its the same as in [Installing a cluster on AWS into an existing VPC](https://53265--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-vpc.html)